### PR TITLE
implement 'hf 15 restore'

### PR DESCRIPTION
**Pull request by brianpow**
implement 'hf 15 restore'

### CODE REVIEW

File: client/cmdhf15.c [hunk 1]
```diff
@@ -754,11 +762,111 @@ int CmdHF15Dump(const char*Cmd) {
...
+	uint8_t cmdp = 0;
+	char newCmdPrefix[255] = {0x00}, tmpCmd[255] = {0x00};
+	char param[FILE_PATH_SIZE]="";
+	char hex[255]="";
+	uint8_t retries = 3, tried = 0;
+	int retval=0;
+	size_t bytes_read;
+	uint8_t i=0;
+		while(param_getchar(Cmd, cmdp) != 0x00) {
+		switch(tolower(param_getchar(Cmd, cmdp))) {
```
Comment (iceman1001) [inline client/cmdhf15.c:780]: good one,   why didn't I think of that...

**Review by iceman1001 — CHANGES_REQUESTED** [review]
Great!  Lovin' it.  
Would you mind adding spaces between..  I really dislike that compressed codestyle...

:)

**Comment by brianpow** [comment]
I found I spent quite some time aligning the space in the help text. A tab will be much easier to maintenance and save our time for new ideas. There are rooms for improving the current parsing algorithm and help text but I have not come up a good idea yet. I am still struggling if there should be a dash before options too.

**Comment by iceman1001** [comment]
aha,  missunderstanding.. I didn't mean the tab in the helptext.  Thats fine.

The commandline parsing is discussed in the offical pm3 repo as we speak,   where a unifying parse with dash and even long parameters is accepted.   It hasn't been impl yet though.

I meant the   crammed   if(x=10);  style,

**Pull request merged**